pkg/genswagger/main.go

```diff
@@ -19,6 +20,22 @@ import (
 
 var outputFile = flag.String("output", "api/openapi-spec/swagger.json", "Output file.")
 
+var kindToTagMappings = map[string]string{
+	"VirtualMachine":                  "Virtual Machines",
+	"VirtualMachineInstance":          "Virtual Machines",
+	"VirtualMachineTemplate":          "Virtual Machine Templates",
+	"VirtualMachineTemplateVersion":   "Virtual Machine Templates",
+	"DataVolume":                      "Volumes",
+	"VirtualMachineImage":             "Images",
+	"VirtualMachineBackup":            "Backups",
```

Comment:
can we please also add `virtualmachinebackupcontents.harvesterhci.io`

Reply:
It is operated by controllers so I think we don't need to add it to the API endpoints